engine/src/main/java/com/datatorrent/stram/StramUtils.java

```diff
@@ -133,4 +135,37 @@ public static JSONObject getStackTrace()
     return jsonObject;
   }
 
+  public static JSONObject getStackTrace2() throws JSONException
+  {
+
+    String result = "hello";
+    try {
+
+      String processName = java.lang.management.ManagementFactory.getRuntimeMXBean().getName();
+      Long processId = Long.parseLong(processName.split("@")[0]);
+
+      String cmd = "jstack " + processId;
```

Comment:
@tushargosavi Thanks for the review. From your experience in debugging customer issues do you think having the following apex cli command option will be a useful addition?

run-jdktools "ContainerId" "Tool name" "Arguments"